src/controllers/MainController.js

```diff
@@ -0,0 +1,66 @@
+const Model = require('../Model/Model');
+const { 
+  readPlayerNum,
+  readRestartEnd,
+  printBallStrike,
+  printBall,
+  printStrike,
+  printNothing,
+  printThreeStrike,
+  exit,
+ } = require('../view/view');
+
+
+class MainController {
+  mainGameController(){
+    let input;
```

Comment:
우리 초기화되지 않은 값을 사용하는건 지양합시다
```suggestion
    let input = null;
```